### PR TITLE
Analyzers update for 8.0

### PR DESCRIPTION
**Pull request by damien-urruty-sonarsource**

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -57,6 +57,7 @@
 
     <change-notes><![CDATA[
       <ul>
+        <li>8.0 - Support of JavaScript analysis in HTML files. Support clang-cl and Microchip compilers in CLion. New quick fix for a C++ rule. New quick fixes for 4 Python rules. 1 new rule for VB.NET in Rider. Bug fixes, fewer FPs and improvements for many languages.</li>
```
Comment (henryju) [inline src/main/resources/META-INF/plugin.xml:60]: For clang-cl support, shouldn't this PR be merged first? https://github.com/SonarSource/sonarlint-intellij/pull/582

Comment (henryju) [reply src/main/resources/META-INF/plugin.xml:60]: We don't support VB.NET in Rider...

**Review by henryju — CHANGES_REQUESTED** [review]

File: src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (damien-urruty-sonarsource) [reply src/main/resources/META-INF/plugin.xml:60]: For 1. yes, I was about to merge it after this one. But I will move the release notes to the other PR
For 2. OK I learned something :smile:

**Review by henryju — APPROVED** [review]
I would change the last commit message (and JIRA ticket) to not mention VB.NET, else LGTM

**Pull request merged**